src/msg/async/frames_v2.h

```diff
@@ -345,6 +357,10 @@ class FrameAssembler {
     return m_crypto->rx->get_extra_size_at_final();
   }
 
+  bool is_compressed() const { return m_flags & FRAME_EARLY_DATA_COMPRESSED; }
```

Comment:
might want to start a new line for the `return` statement for better readability.